modules/configuring-quota-management-for-distributed-workloads.adoc

```diff
@@ -38,13 +38,24 @@ For a complete list of the default notebook images and their preinstalled packag
 endif::[]
 
 * You have sufficient resources. In addition to the base {productname-short} resources, you need 1.6 vCPU and 2 GiB memory to deploy the distributed workloads infrastructure.
+
+ifndef::upstream[]
+* The resources are physically available in the cluster.
++
+[NOTE]
+====
+{org-name} currently supports only a single cluster queue per cluster (that is, homogenous clusters), and only empty resource flavors.
```

Comment:
is {org-name} here "Red Hat" or "OpenDataHub"?

Reply:
{org-name} = Red Hat